public/js/app.js

```diff
@@ -4771,9 +4771,11 @@ function initQuickLinksForm() {
 }
 
 function showFilterField() {
-  selector = $('#entity-selector');
+  var selector = $('#entity-selector');
```

Comment:
Since selector is only used in this function, `let` is the correct declarator, since we have a `let selector` in other functions in the same script.